src/redisai.c

```diff
@@ -578,6 +578,7 @@ struct RedisAI_RunInfo {
   RedisModuleString **outkeys;
   RAI_ModelRunCtx* mctx;
   int status;
+  RAI_Error* err;
```

Comment:
Why a pointer member?

Reply:
Right, that could indeed be the object itself. However, `RAI_Error` contains strings (for the error message) that would have to be copied on the heap anyway, so in the end I felt it was more cumbersome. Let me know if you think differently.